code/modules/power/cable.dm

```diff
@@ -565,15 +565,19 @@ obj/structure/cable/proc/cableColor(var/colorC)
 		w_class = ITEMSIZE_SMALL
 
 /obj/item/stack/cable_coil/examine(mob/user)
-	if(get_dist(src, user) > 1)
-		return
+	var/msg = ""
 
 	if(get_amount() == 1)
-		to_chat(user, "A short piece of power cable.")
+		msg += "A short piece of power cable."

```

Comment:
Just a question- what's the difference between this and to_chat?

Reply:
In this instance, msg is a string or text bar that gets modified based on how much cable is in the coil, and if you're close enough it adds in the amount of said amount is greater than 2, then passes the msg string to the user through a to_chat().